kfs-purap/src/main/resources/edu/arizona/kfs/module/purap/spring-purap.xml

```diff
@@ -103,5 +108,13 @@
             </list>
         </property>
     </bean>
+    
+    <bean id="printService" parent="printService-parentBean"  class="edu.arizona.kfs.module.purap.document.service.impl.PrintServiceImpl">
```

Comment:
Why is the edu.arizona version of this service impl needed? Lets discuss the best way to implement the required PDF changes.